cpp/include/Ice/Communicator.h

```diff
@@ -199,129 +186,121 @@ namespace Ice
         /// @see Connection::setAdapter
         void setDefaultObjectAdapter(ObjectAdapterPtr adapter);
 
-        /// Get the implicit context associated with this communicator.
-        /// @return The implicit context associated with this communicator; returns null when the property
-        /// Ice.ImplicitContext is not set or is set to None.
+        /// Gets the implicit context associated with this communicator.
+        /// @return The implicit context associated with this communicator; returns `nullptr` when the property
+        /// `Ice.ImplicitContext` is not set or is set to None.
```

Comment:
Maybe quote "None" to make clear is a property value.

```suggestion
        /// `Ice.ImplicitContext` is not set or is set to "None".
```